bobtemplates/django/+package.fullname+/README.md.bob

```diff
@@ -0,0 +1,124 @@
+   createdb deployment_database
+   ```
+
+3. Add deployment scripts to repository:
+   ```
+   wget https://raw.githubusercontent.com/4teamwork/django-git-deployment/master/deploy/setup.sh
+   chmod +x setup.sh
+   ./setup.sh
+   ```
+
+4. Initialize the git repository in the deployment directory on the server:
+   ```
+   git deploy setup -r deployment_name
+   ```
+
+5. Push the code:
+   ```
+   git push deployment_name master
+   ```
+
+6. Add the environment variables needed by the Django backend:
+   ```
+   ssh -t user@example.com "cd /path/to/desired/project/folder && exec \$SHELL -l"
+   cp .env.template .env
+   vim .env
+   ```
+
+   These variables can / need to be set:
+   - `DJANGO_ALLOWED_HOSTS`: [A list of strings representing the host/domain names that this Django site can serve.](https://docs.djangoproject.com/ko/1.11/ref/settings/#allowed-hosts)
+   - `DJANGO_DATABASE_ENGINE`: [The database engine to connect to the database.](https://docs.djangoproject.com/ko/1.11/ref/settings/#engine) Defaults to `django.db.backends.postgresql_psycopg2`.
+   - `DJANGO_DATABASE_HOST`: [Which host to use when connecting to the database.](https://docs.djangoproject.com/ko/1.11/ref/settings/#host)
+   - `DJANGO_DATABASE_NAME`: [The name of the database to use.](https://docs.djangoproject.com/ko/1.11/ref/settings/#name)
+   - `DJANGO_LOGFILE`: The path to the logfile. Defaults to `log/info.log` in the project's root folder.
+   - `DJANGO_LOGGING_LEVEL_{{{package.fullname|upper}}}`: The logging level for the {{{package.fullname}}} application. Defaults to `WARNING`.
+   - `DJANGO_LOGGING_LEVEL_DJANGO`: The logging level for the Django. Defaults to `WARNING`.
+   - `DJANGO_LOGGING_LEVEL_ROOT`: The logging level for everything else. Defaults to `WARNING`.
+   - `DJANGO_MEDIA_ROOT`: The folder containing the application's files. Defaults to `media` in the project's root folder.
+   - `DJANGO_MEDIA_URL`: The URL for the media. Defaults to `/media/`.
+   - `DJANGO_MODE`: The mode in which you want the backend to run. [Read `django-classy-settings` for more information.](https://django-classy-settings.readthedocs.io/)
+   - `DJANGO_RAVEN_DSN`: The DSN for the project in sentry.
+   - `DJANGO_SECRET_KEY`: [A secret key for a particular Django installation.](https://docs.djangoproject.com/ko/1.11/ref/settings/#secret-key)
+   - `DJANGO_SESSION_COOKIE_AGE`: [The age of the session cookies, in seconds.](https://docs.djangoproject.com/en/1.11/ref/settings/#session-cookie-age)
+   - `DJANGO_SETTINGS_MODULE`: [The settins module for the Django backend.](https://docs.djangoproject.com/en/1.11/topics/settings/#envvar-DJANGO_SETTINGS_MODULE)
+   - `DJANGO_ZIPFILES_ROOT`: The folder where the zipfiles are stored prior to download. Defaults to `zipfiles` in the project's media folder.
+
+   Some of these values are preset.
+   Complete the file by adding the missing values.
+
+   Finally run `after_push`:
+   ```
+   source deploy/after_push
+   ```
```

Comment:
Maybe it would be good to also add a section about how to deploy updates onto an already installed deployment. It may not be obvious that this is done with git push.
Especially also document how to push a local branch ( `git push production branch:master`)